src/main/java/net/earthcomputer/clientcommands/command/AliasCommand.java

```diff
@@ -0,0 +1,186 @@
+
+    private static final SimpleCommandExceptionType FILE_READ_EXCEPTION = new SimpleCommandExceptionType(new TranslatableText("commands.calias.file.readError"));
+    private static final SimpleCommandExceptionType FILE_WRITE_EXCEPTION = new SimpleCommandExceptionType(new TranslatableText("commands.calias.file.writeError"));
+    private static final SimpleCommandExceptionType TOO_FEW_ARGUMENTS_EXCEPTION = new SimpleCommandExceptionType(new TranslatableText("commands.calias.tooFewArguments"));
+
+    private static final DynamicCommandExceptionType ALIAS_EXISTS_EXCEPTION = new DynamicCommandExceptionType(arg -> new TranslatableText("commands.calias.addAlias.aliasAlreadyExists", arg));
+    private static final DynamicCommandExceptionType COMMAND_EXISTS_EXCEPTION = new DynamicCommandExceptionType(arg -> new TranslatableText("commands.calias.addAlias.commandAlreadyExists", arg));
+    private static final DynamicCommandExceptionType NOT_FOUND_EXCEPTION = new DynamicCommandExceptionType(arg -> new TranslatableText("commands.calias.notFound", arg));
+
+    private static HashMap<String, String> aliasMap = new HashMap<>();
+
+    static {
+        try {
+            getAliases();
+        } catch (Exception e) {
+            LOGGER.error("No alias file provided. A new one will be created upon registering an alias.", e);
+        }
+    }
+
+    public static void register(CommandDispatcher<ServerCommandSource> dispatcher) {
+        addClientSideCommand("calias");
+
+        dispatcher.register(literal("calias")
+                .then(literal("add")
+                        .then(argument("key", string())
+                                .then(argument("command", greedyString())
+                                        .executes(ctx -> addAlias(getString(ctx, "key"), getString(ctx, "command"))))))
+                .then(literal("list")
+                        .executes(ctx -> listAliases()))
+                .then(literal("remove")
+                        .then(argument("key", string())
+                                .executes(ctx -> removeAlias(getString(ctx, "key"))))));
+
+        for (String key: aliasMap.keySet()) {
+            if (dispatcher.getRoot().getChildren().stream().map(CommandNode::getName).noneMatch(literal -> literal.equals(key))) {
+                addClientSideCommand(key);
+                dispatcher.register(literal(key)
+                        .executes(ctx -> executeAliasCommand(key, null))
+                        .then(argument("arguments", greedyString())
+                                .executes(ctx -> executeAliasCommand(key, getString(ctx, "arguments")))));
+            } else {
+                LOGGER.error("Attempted to register alias /{}, but that command already exists", key);
+            }
+        }
+    }
+    private static int executeAliasCommand(String aliasKey, String arguments) throws CommandSyntaxException {
+
```

Comment:
Not usually this picky about formatting but this really bugs me. Please no blank lines at the start of methods